src/com/spikes2212/genericsubsystems/limitationFunctions/Limitless.java

```diff
@@ -0,0 +1,36 @@
+package com.spikes2212.genericsubsystems.limitationFunctions;
+
+import java.util.function.Function;
+
+import com.spikes2212.genericsubsystems.BasicSubsystem;
+
+/**
+ * This is a function from double to boolean that always returns true. This is
+ * for a {@link BasicSubsystem} with no limits.
+ * 

```

Comment:
make it very clear that we use this as the function in the basicSubsystem constructor